api/src/main/java/io/opentelemetry/trace/DefaultSpan.java

```diff
@@ -18,33 +18,42 @@
 
 import io.opentelemetry.internal.Utils;
 import java.util.Map;
+import java.util.Random;
 import javax.annotation.concurrent.Immutable;
 
 /**
- * The {@code BlankSpan} is the default {@link Span} that is used when no {@code Span}
+ * The {@code DefaultSpan} is the default {@link Span} that is used when no {@code Span}
  * implementation is available. All operations are no-op except context propagation.
  *
- * <p>When no valid context ({@code null} or {@link SpanContext#BLANK}) is available then
- * implementation propagates {@link BlankSpan#INSTANCE} which encapsulates {@link
- * SpanContext#BLANK}.
- *
  * <p>Used also to stop tracing, see {@link Tracer#withSpan}.
  *
  * @since 0.1.0
  */
 @Immutable
-public final class BlankSpan implements Span {
+public final class DefaultSpan implements Span {
+
+  private static final Random random = new Random();
+
   /**
-   * An instance of this class. If there is no {@code SpanContext} or {@link SpanContext#BLANK} to
-   * propagate this instance is used.
+   * Creates an instance of this class.
    *
+   * <p>Each instance will have a unique and valid {@link SpanId}.
+   *
+   * @return a {@link DefaultSpan}.
    * @since 0.1.0
    */
-  public static final Span INSTANCE = new BlankSpan(SpanContext.BLANK);
+  public static DefaultSpan create() {
+    return new DefaultSpan(
+        SpanContext.create(
+            TraceId.INVALID,
+            SpanId.generateRandomId(random),
```

Comment:
Don't think the expected behavior is to have only a valid SpanId but invalid TraceId. Please revert this and keep this PR minimal (only rename the class).

Reply:
https://github.com/open-telemetry/opentelemetry-java/issues/310#issuecomment-498797101 only mentioned `SpanId` and I'm less sure about `TraceId`. Reverted other changes for now.